cli/src/pcluster/cli_commands/commands.py

```diff
@@ -296,6 +288,18 @@ def status(args):
         sys.exit(0)
 
 
+def _print_cluster_status(cluster_name, scheduler):
+    """Print head node and compute fleet status."""
+    result = PclusterApi().describe_cluster_instances(
+        cluster_name=cluster_name, region=get_region(), node_type=NodeType.HEAD_NODE
+    )
+    if isinstance(result, list) and len(result) == 1:
+        LOGGER.info("%s: %s", result[0].node_type, result[0].state)
+        _print_compute_fleet_status(cluster_name, scheduler)
+    else:
+        utils.error(f"Unable to retrieve the status of the cluster's instances.\n{result.message}")
```

Comment:
Is it possible for `result.message` to be an empty list here? Asking because the condition checks that result is a list and has at least one entry.

Reply:
This shouldn't be possible, but anyway improved, thanks!